pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -547,11 +548,15 @@ type BootMenu struct {
 }
 
 // TODO <loader readonly='yes' secure='no' type='rom'>/usr/lib/xen/boot/hvmloader</loader>
-type BIOS struct {
+type Loader struct {
+	ReadOnly bool   `xml:"readonly,attr,omitempty"`
+	Secure   bool   `xml:"secure,attr,omitempty"`
+	Type     string `xml:"type,attr,omitempty"`
+	Path     string `xml:",chardata,omitempty"`
```

Comment:
In testing, I found that this actually should be:

```suggestion
	Path     string `xml:",chardata"`
```